app/views/movies/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2> Movie Show Page </h2>
+
+<%= @movie.title %>
+<%= @movie.creation_year %>
+<%= @movie.genre %>
+
+<p> Actors: </p>
+
+<p><% @actors.each do |actor| %>
+  <li>Name: <%= actor.name %>
+  Age: <%= actor.age %></li>
+  <% end %></p>
+  <p> Actors Average Age: <%= @actors.average_age %></p>
+
+
+  <h3> Add a Actor to this Movie: </h3>
+  
+  <%= form_with url: "/movies/#{@movie.id}?actor_name=#{params[:name]}&commit=Search", method: :get do %>
```

Comment:
You don't need to use query parameters here if you use a route that can carry the movie ID, and then keep in mind that the form field for the actor (which should be ID, not name!) will have the actor info so that doesn't need to be in this url either. 
`form_with url: "/movies/:movie_id/actors", method: :post, data: {turbo: false} do |form|`

This is something I remember from your pair project as well, so I encourage you to refactor it in this IC or in your project (or both!) and add a note about this in a cheat sheet so that you don't continue to format forms like this!